references/docgen/def-doc/trait/container-ports.eg.md

```diff
@@ -0,0 +1,30 @@
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: busybox
+spec:
+  components:
+  - name: busybox
+    properties:
+      cpu: "0.5"
+      exposeType: ClusterIP
+      image: busybox
+      memory: 1024Mi
+      ports:
+      - expose: false
+        port: 80
+        protocol: TCP
+      - expose: false
+        port: 801
+        protocol: TCP
```

Comment:
```yaml
- expose: false
   port: 801
   protocol: TCP
   nodePort: 8080
```
Is this more clear and simple?

Reply:
@barnettZQG  `ports` It doesn't have a nodePort attribute

Reply:
https://github.com/kubevela/kubevela/blob/9203bbcdb8c79e9e44bc6eb9072ad477ee65be75/vela-templates/definitions/internal/component/webservice.cue#L409

Reply:
@barnettZQG the container-ports trait applies only when webservice.expose is false, and use `hostPort`  to expose.